--- sqliteodbc-0.996/Makefile.in.orig	2014-02-25 12:26:51.932493643 +0100
+++ sqliteodbc-0.996/Makefile.in	2014-02-25 15:07:30.478755818 +0100
@@ -134,7 +134,10 @@
 		    -I$(SQLITE3_INC) \
 		    $(SQLITE3_FLAGS) $(XML2_FLAGS) xpath.c
 
-install-2:	libsqliteodbc.la
+$(DESTDIR)$(drvdir):
+	install -d $(DESTDIR)$(drvdir)
+
+install-2:	libsqliteodbc.la $(DESTDIR)$(drvdir)
 		$(LIBTOOL) --mode=install $(INSTALL) \
 		    libsqliteodbc.la $(DESTDIR)$(drvdir)
 
@@ -150,7 +153,7 @@
 		    $(RM) $(DESTDIR)$(drvdir)/libsqliteodbc.la
 
 install-3:	libsqlite3odbc.la @EXT_BLOBTOXY@ @EXT_IMPEXP@ \
-		@EXT_CSVTABLE@ @EXT_ZIPFILE@ @EXT_XPATH@
+		@EXT_CSVTABLE@ @EXT_ZIPFILE@ @EXT_XPATH@ $(DESTDIR)$(drvdir)
 		$(LIBTOOL) --mode=install $(INSTALL) \
 		    libsqlite3odbc.la $(DESTDIR)$(drvdir)
 		test -z "@EXT_BLOBTOXY@" || \
@@ -196,7 +199,7 @@
 		sh drvdsnuninst.sh SQLITE3 "SQLite3 Datasource"
 
 
-install-4:	libsqlite4odbc.la
+install-4:	libsqlite4odbc.la $(DESTDIR)$(drvdir)
 		$(LIBTOOL) --mode=install $(INSTALL) \
 		    libsqlite4odbc.la $(DESTDIR)$(drvdir)
 
